.bundle/config

```diff
@@ -1,5 +1,6 @@
 ---
-# Use a cache - save rubygems bandwidth
 BUNDLE_CACHE_ALL: "true"
-# Nokogiri should _behave_
-BUNDLE_FORCE_RUBY_PLATFORM: "true"
\ No newline at end of file
+BUNDLE_FORCE_RUBY_PLATFORM: "true"
+BUNDLE_PATH: "/home/runner/work/MeetAnotherDay/MeetAnotherDay/vendor/bundle"
+BUNDLE_DEPLOYMENT: "true"
+BUNDLE_JOBS: "4"
```

Comment:
Revert any changes to .bundle/config. There should be no changes to this file in the final PR.